src/py/migraphx_py.cpp

```diff
@@ -612,6 +613,8 @@ MIGRAPHX_PYBIND11_MODULE(migraphx, m)
 #ifdef VERSION_INFO
     m.attr("__version__") = VERSION_INFO;
 #else
-    m.attr("__version__") = "dev";
+    m.attr("__version__") = std::to_string(MIGRAPHX_VERSION_MAJOR) + "." +
```

Comment:
Inside `migraphx-driver` it also prints SHA with `MIGRAPHX_VERSION_TWEAK`
It has been useful for debugging.  We can consider adding that but that would be deviation from https://packaging.python.org/en/latest/discussions/versioning/

Reply:
You can also do something like “0.5.dev1+gd00980f”
https://packaging.python.org/en/latest/discussions/versioning/#local-version-identifiers

Reply:
oh ya, I'll update it to add tweak in that form